src/Components/ArtworkFilter/Dropdown.tsx

```diff
@@ -23,7 +23,7 @@ interface DropdownState {
 }
 
 export class Dropdown extends React.Component<DropdownProps, DropdownState> {
-  constructor(props) {
+  constructor(props: DropdownProps) {
```

Comment:
When overriding the `constructor`, the props types have to explicitly be declared again, because this is the signature that the JSX usage of the component is based on.